src/main/java/uk/gov/digital/ho/hocs/application/LogEvent.java

```diff
@@ -18,7 +18,8 @@ public enum LogEvent {
     REST_HELPER_GET,
     REST_HELPER_POST,
     REST_HELPER_PUT,
-    TOO_MANY_MESSAGES;
+    TOO_MANY_MESSAGES,
+    DOCUMENT_CLIENT_CREATE_SUCCESS;
```

Comment:
Nitpick: Trying to keep these in alphabetical order to improve cognitive complexity - could this be moved please?

Reply:
Done.